web/src/testgrid-dashboard-summary.ts

```diff
@@ -28,7 +29,7 @@ const tabStatusIcon = new Map<string, string>([
 ]);
 
 // TODO: generate the correct time representation
-function convertResponse(ts: TabSummary) {
+function convertResponse(ts: TabSummary, dbName: string) {
```

Comment:
I believe this was not working correctly because our fake-server was not providing correct dashboard name - the fix is available in https://github.com/GoogleCloudPlatform/testgrid/pull/1216. This is fine for now